recipes/nbpresent/post-link.bat

```diff
@@ -0,0 +1 @@
+"%PREFIX%\Scripts\jupyter-nbextension.exe" enable nbpresent --py --sys-prefix && "%PREFIX%\Scripts\jupyter-serverextension.exe" enable --py nbpresent --sys-prefix && if errorlevel 1 exit 1

```

Comment:
I might do the super-aggressive, always fail thing.... since we're on the, topic, is there an equivalent of `-e`?